internal/metadata/metadata_test.go

```diff
@@ -84,3 +88,53 @@ func TestSet(t *testing.T) {
 		})
 	}
 }
+
+func TestValidate(t *testing.T) {
+	for _, test := range []struct {
+		md        metadata.MD
+		want      error
+	}{
+		{
+			md:        Pairs(string(rune(0x19)), "testVal"),
```

Comment:
You don't need to call `Pairs` to make md. Just convert from `map` to `metadata`.
Then you can get rid of the function `Pairs` in this file.

Reply:
I found a function `metadata.Pairs`